### PR TITLE
AE-1859: Sentinel and Autotask for V3 Liquidatable Positions Bot

### PR DESCRIPTION
**Pull request by Nomad-AE**
Added Sentinel, Autotask, and test

### CODE REVIEW

File: defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  const { alert } = body;
+  if (alert === undefined) {
+    throw new Error('alert undefined');
+  }
+
+  // ensure that the alert key exists within the body Object
+  const { source } = body;
+  if (source === undefined) {
+    throw new Error('source undefined');
+  }
```
Comment (teasipper-ae) [inline defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.js:82]: Combine into 1 destructuring statement.
```suggestion
const { alert, source } = body;
if (alert === undefined) {
  throw new Error('alert undefined');
} else if (source  === undefined) {
  throw new Error('source undefined');
}
```

File: defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  if (metadata === undefined) {
+    throw new Error('metadata undefined');
+  }
+
+  const {
+    borrowerAddress,
+    blockNumber,
+  } = metadata;
+
+  const message = `📉💵🔥 **Comet V3 Liquidatable account detected** account ${borrowerAddress.slice(0, 6)} `
```
Comment (teasipper-ae) [inline defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.js:95]: In my v2/v3 tagging branch, I added this to the end of all appropriate messages: `(Compound v3)`.  So your message would look like this: `📉💵🔥 **Liquidatable account detected** account 0x010011010101 (Compound v3)` .  For consistency, should stick with this format?

File: defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.spec.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  protocol: 'Protocol',
+  metadata: mockMetadata,
+});
+
+// eslint-disable-next-line import/no-useless-path-segments
+const { handler } = require('../downloaded/Forta_CompV3_Liquidation_Monitor');
+
+function createFortaSentinelEvent(finding, blockHash) {
+  // Generally findings go from the Bot, to Scan Node, to Sentinel, to Autotasks
+  //  with some metadata being added and removed along the way. This function will mimic
```
Comment (echoAE) [inline defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.spec.js:47]: Mega nit.
```suggestion
  // with some metadata being added and removed along the way. This function will mimic
```

**Review by echoAE — APPROVED** [review]

File: defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.js (hunk 2 above)
Comment (Nomad-AE) [reply defender/staged/downloaded/Forta_CompV3_Liquidation_Monitor.js]: Fixed in b7f0cf831ea1036e4a668c9f0f78f00f3e0c33d4

**Review by teasipper-ae — APPROVED** [review]

**Pull request merged**